src/main/java/com/opentok/OpenTok.java

```diff
@@ -61,15 +61,15 @@
      * @param apiSecret Your OpenTok API secret. (See the <a href="https://dashboard.tokbox.com">OpenTok dashboard</a>
      * page.)
      */
-    public OpenTok(int apiKey, String apiSecret) {
+    private OpenTok(int apiKey, String apiSecret) {
         this(apiKey, apiSecret, defaultApiUrl);
     }
 
-    public OpenTok(int apiKey, String apiSecret, String apiUrl) {
+    private OpenTok(int apiKey, String apiSecret, String apiUrl) {
```

Comment:
This and the following private constructor are not necessary. The Builder class has access to the OpenTok instance's private members, so it could just use the one public constructor and then assign its `apiUrl` and/or `proxy` properties.